business_requirement_deliverable_project/wizard/generate_projects.py

```diff
@@ -137,8 +137,9 @@ def generate_br_projects(self, parent_project, br, project_ids, task_ids):
         if self.for_childs:
             br_project = br_project or parent_project
             for child_br in br.business_requirement_ids:
-                self.generate_br_projects(
-                    br_project, child_br, project_ids, task_ids)
+                if child_br.state == 'stakeholder_approval':
```

Comment:
@pedrobaeza Yes: actually that is the point to allow the generation of the projects  in multiple steps (not all BR are ready at the same time).
Requirement is to be able to launch the wizard any time as the button is not made invisible until all projects are generated